hexa/pipelines/permissions.py

```diff
@@ -37,3 +37,13 @@ def run_pipeline(principal: User, pipeline: Pipeline):
             user=principal,
         ).exists()
     )
+
+
+def delete_pipeline_version(principal: User, pipeline: Pipeline):
```

Comment:
- the `pipeline.workspace` check is not needed
- allow editors to delete versions as well (if not too complex, only their own versions)